Docs v2 (#79)
apify/apify-sdk-python: docs/03-concepts/03-storages.mdx

which function as a read/write storage for storing file-like objects, typically the actor state or binary results.
You can open a key-value store through the [`Actor.open_key_value_store()`](../../reference/class/Actor#open_key_value_store) method,
and work with it through the resulting [`KeyValueStore`](../../reference/class/KeyValueStore) class instance.

Finally, there are [request queues](https://docs.apify.com/platform/storage/request-queue),
which are queues into which you can put URLs that you want to scrape,
and from which the actor can dequeue them and process them.
You can open a request queue through the [`Actor.open_request_queue()`](../../reference/class/Actor#open_request_queue) method,
and work with it through the resulting [`RequestQueue`](../../reference/class/RequestQueue) class instance.

Each actor run has its default dataset, default key-value store and default request queue.

## Local storage emulation

To be able to develop actors locally,
the storages that the Apify Platform provides are emulated on the local filesystem.

The storage contents are loaded from and saved to the `storage` folder in the actor's main folder.

> Each type of storages is stored in its own subfolder, so e.g. datasets are stored in the `storage/datasets` folder.

Reviewer: ```suggestion
Each storage type is stored in its own subfolder, so datasets, for example, are stored in the `storage/datasets` folder.
```
